src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -440,7 +439,6 @@ def _create_pgm_input_lines(self):
         pgm_lines["r0"] = self._get_pp_attr("line", "r0_ohm_per_km", expected_type="f8", default=np.nan) * multiplier
         pgm_lines["x0"] = self._get_pp_attr("line", "x0_ohm_per_km", expected_type="f8", default=np.nan) * multiplier
         pgm_lines["c0"] = c0_nf_per_km * length_km * parallel * 1e-9
-        pgm_lines["tan0"] = 0.0
```

Comment:
idem:

```py
invalid = np.isclose(c0_nf_per_km, 0.0)
pgm_lines["tan0"] = np.divide(
    self._get_pp_attr("line", "g0_us_per_km", expected_type="f8", default=0),
    c0_nf_per_km * (2 * np.pi * self.system_frequency * 1e-3),
    where=np.logical_not(invalid),
)
pgm_lines["tan0"][invalid] = 0.0
```

Reply:
nvm. just read https://github.com/PowerGridModel/power-grid-model-io/pull/326#issuecomment-3279663612 again. Resolving

> @mgovers @nitbharambe there is still a reliability issue on pypi.
I've checked that 2 No. lines causing this issue can be removed, the field is auto initialised with zero even when np.divide has some False entries in where.